### PR TITLE
Added new menu "Contribute" and "Resources"

### PR DESCRIPTION
**Pull request by ravage84**
Adjusted menu items accordingly

Added menu item to the new translations platform on https://www.transifex.com/projects/p/cakephp/

I'm not 100 % sure about the order of some items but I'm happy to hear suggestions.

### CODE REVIEW

File: views/elements/navbar.ctp [hunk 1]
```diff
@@ -28,25 +28,37 @@
 				</li>
 			</ul>
 		</li>
+		<li class="primary"><a href="#" class="empty">Contribute</a>
+			<ul class="second-level">
+				<li><a href="http://book.cakephp.org/2.0/en/contributing.html">How to Contribute</a></li>
+				<li><a href="https://github.com/cakephp/">GitHub</a></li>
+				<li><a href="http://github.com/cakephp/cakephp/contributors">Contributors</a></li>
```
Comment (ADmad) [inline views/elements/navbar.ctp]: I would move this last. It doesn't really help others in contributing, more for informative purpose.

Comment (ravage84) [inline views/elements/navbar.ctp]: OK, will change that. I wanted the main page or better said its menu to be more inviting for potential contributors...

File: views/elements/navbar.ctp [hunk 2]
```diff
@@ -28,25 +28,37 @@
 				</li>
 			</ul>
 		</li>
+		<li class="primary"><a href="#" class="empty">Contribute</a>
+			<ul class="second-level">
+				<li><a href="http://book.cakephp.org/2.0/en/contributing.html">How to Contribute</a></li>
+				<li><a href="https://github.com/cakephp/">GitHub</a></li>
```
Comment (jameswatts) [inline views/elements/navbar.ctp]: Maybe change the text of this option to "Source Code", "Repository" or something more descriptive?

File: views/elements/navbar.ctp [hunk 3]
```diff
@@ -28,25 +28,37 @@
 				</li>
 			</ul>
 		</li>
+		<li class="primary"><a href="#" class="empty">Contribute</a>
+			<ul class="second-level">
```
Comment (jameswatts) [inline views/elements/navbar.ctp:37]: "Roadmap" would be a good option to add somewhere here, it gets asked for quite regularly.

File: views/elements/navbar.ctp [hunk 4]
```diff
@@ -28,25 +28,37 @@
...
+				<li><a href="http://book.cakephp.org/2.0/en/contributing.html">How to Contribute</a></li>
```
Comment (jameswatts) [inline views/elements/navbar.ctp:39]: Would http://cakephp.org/development not be a more adequate page to link to here?

Comment (ceeram) [inline views/elements/navbar.ctp:39]: @James, http://cakephp.org/development is not just about contributing, and
the contribution section just links to the proposed page.

2013/7/26 James Watts notifications@github.com

Comment (jameswatts) [inline views/elements/navbar.ctp:39]: Then I think at least a link to something regarding security would be good, as there is on that page. Surely you're seeing the same as me, that we have a lot of this information repeated all over the place.

File: views/elements/navbar.ctp (hunk 3 above)
Comment (ravage84) [inline views/elements/navbar.ctp:37]: Great idea, thanks!

Would you link to something like Debug_Kit's roadmap - just for CakePHP (seems there is none)?
https://cakephp.lighthouseapp.com/projects/42880/roadmap
Or is there a road map page in particular somewhere else?

File: views/elements/navbar.ctp (hunk 4 above)
Comment (ravage84) [inline views/elements/navbar.ctp:39]: Personally I'm thinking about the whole information fragmentation quite a a bit lately.
What if we move those few pages like http://cakephp.org/development into the book?
That way the book could become not only the real one stop for information but also those pages become ediatable through GitHub and the book becomes more self-containing.

When done, pages like http://cakephp.org/development just link to the appropriate page in the book.

I'm always thinking of the Cookbook as if it was printed and handed out. Wouldn't it be great if you would have the holy bible of CakePHP info in your hand?

File: views/elements/navbar.ctp (hunk 2 above)
Comment (ravage84) [inline views/elements/navbar.ctp]: Yeah I wasn't sure myself.

What about "Source Code on GitHub", "Source Code @ GitHub", "GitHub Repository" or may be "Source Code Repository"?

File: views/elements/navbar.ctp (hunk 3 above)
Comment (ADmad) [inline views/elements/navbar.ctp:37]: This is a better one https://cakephp.lighthouseapp.com/projects/42648/development-roadmap  though even that isn't keep upto date reguarly

**Comment by ravage84** [comment]
Wouldn't be http://cakephp.org/changelogs a good addition to the Downloads menu?

File: views/elements/navbar.ctp (hunk 1 above)
Comment (ravage84) [inline views/elements/navbar.ctp]: Nevermind my comment. I mistook you meant to move the whole Contribute menu item to the end... But the contributors list is certainly only a nice to have info....

File: views/elements/navbar.ctp (hunk 3 above)
Comment (ravage84) [inline views/elements/navbar.ctp:37]: done.

File: views/elements/navbar.ctp (hunk 2 above)
Comment (ravage84) [inline views/elements/navbar.ctp]: Opted for "Source Code Repositories @ GitHub". A bit long but should make it clear what it is. Also it's a link to all the repos, not just the core.

**Comment by ravage84** [comment]
Are we good to go?

File: views/elements/navbar.ctp [hunk 5]
```diff
@@ -28,25 +28,38 @@
 				</li>
 			</ul>
 		</li>
+		<li class="primary"><a href="#" class="empty">Contribute</a>
+			<ul class="second-level">
+				<li><a href="https://cakephp.lighthouseapp.com/projects/42648/development-roadmap">Roadmap</a></li>
+				<li><a href="http://book.cakephp.org/2.0/en/contributing.html">How to Contribute</a></li>
+				<li><a href="https://github.com/cakephp/">Source Code Repositories @ GitHub</a></li>
```
Comment (jameswatts) [inline views/elements/navbar.ctp]: This text for this menu option is too long, I would suggest reducing it to "Source Code on GitHub", or just "Source Code".

File: views/elements/navbar.ctp [hunk 6]
```diff
@@ -28,25 +28,38 @@
...
+			</ul>
+		</li>
 		<li class="primary"><a href="http://community.cakephp.org">Community</a>
 			<ul class="second-level">
-				<li><a href="http://webchat.freenode.net/?channels=cakephp">Help &amp; Support</a></li>
-				<li><a href="http://cakephp.lighthouseapp.com">Issues</a></li>
-				<li><a href="http://bakery.cakephp.org">The Bakery</a></li>
+				<li><a href="http://community.cakephp.org/get-involved">Get Involved</a></li>
+				<li><a href="http://webchat.freenode.net/?channels=cakephp">Help &amp; Support (IRC)</a></li>
+				<li><a href="http://cakephp.org/news">News</a></li>
```
Comment (jameswatts) [inline views/elements/navbar.ctp]: I would put this option before "Get Involved". The #cakephp IRC channel and Stack Overflow are the main locations for community driven support, so they should be together in the menu, and as close to the top as possible.

**Comment by jameswatts** [comment]
Just tested this with a 10" tablet in portrait and the menu goes off screen to the right. Considering the new "Resources" option only has 2 options in it I think we can leave them under "Community" for now.

**Comment by ravage84** [comment]
@jameswatts I'm sorry but I have to disagree.

First of all half of the point of this PR was to tidy up the Community menu item, because there's already too much in that list.

Second the world is not just made of tablets (even though I have one, too).
I know we should keep the compatibility for all devices at best, but the main device should still be the PC.

Reworking the menu to collapse if the viewport is too narrow would be the better solution.
Just like Twitter Bootstrap does.
Probably there are other solutions that could work here...

File: views/elements/navbar.ctp (hunk 5 above)
Comment (ravage84) [inline views/elements/navbar.ctp]: "Source Code" shall it be. Who cares about where the repo is at the moment of clicking? ;-)

File: views/elements/navbar.ctp (hunk 6 above)
Comment (ravage84) [inline views/elements/navbar.ctp]: True, will do.

**Comment by ravage84** [comment]
@jameswatts Done, please review.

**Comment by jameswatts** [comment]
The menu does actually collapse to a select when on mobile, but it doesn't do the same for tablet in portrait (at least for me on a Nexus 10).
- "I know we should keep the compatibility for all devices at best, but the main device should still be the PC."

Not sure about the others, but I can't +1 a PR which introduces an issue which wasn't there to begin with.

Maybe you could check out why the select doesn't show?

**Comment by ravage84** [comment]
@jameswatts Updated this old PR:

https://github.com/ravage84/csfnavbar/commit/b6fb82becc6ebfccfb8cdeb34c718eeb46b67c39
https://github.com/ravage84/csfnavbar/commit/43eb7de95b490ba51be9a175b5ffbcf643e17584
https://github.com/ravage84/csfnavbar/commit/6295f844eb542b319c9c63b8438bcc3059b21f37
https://github.com/ravage84/csfnavbar/commit/de407a8e31b22fb08ea607e5ae00cf39d09eb97e

The last one could fix your mentioned problem
https://github.com/cakephp/csfnavbar/pull/2#issuecomment-22638655

But the curious thing is the select input which appears when you make your browser windows smaller isn't part of the app itself:
https://github.com/cakephp/csfnavbar/blob/master/views/elements/navbar.ctp

Do you know how it gets created?
Or who should know?

File: views/elements/navbar.ctp [hunk 7]
```diff
@@ -28,22 +33,35 @@
 				</li>
 			</ul>
 		</li>
+		<li class="primary"><a href="#" class="empty">Contribute</a>
+			<ul class="second-level">
+				<li><a href="https://cakephp.lighthouseapp.com/projects/42648/development-roadmap">Roadmap</a></li>
```
Comment (ADmad) [inline views/elements/navbar.ctp]: Need to get rid of this. Can add links to the 2.6, 3.0 roadmaps on GH wiki.

Comment (ravage84) [inline views/elements/navbar.ctp]: Done, thanks

**Comment by ravage84** [comment]
@ADmad I'm sorry! I understood your comment

> Can add links to the 2.6, 3.0 roadmaps on GH wiki.

as

> **I** can add links to the 2.6, 3.0 roadmaps on GH wiki.

**Comment by ravage84** [comment]
Withdrawing this PR as I think the whole navigation needs to be reworked. Especially since the new (mobile) design, see:
https://github.com/cakephp/docs/issues/2556

**Pull request closed without merging**